app/controllers/email/VerifyEmailController.scala

```diff
@@ -17,29 +17,25 @@
 package controllers.email
 
 import audit.AuditingService
-import audit.models.ChangedContactPrefEmailAuditModel
 import common.SessionKeys
 import config.{AppConfig, ErrorHandler}
 import controllers.predicates.AuthPredicateComponents
 import controllers.BaseController
 import controllers.predicates.inflight.InFlightPredicateComponents
 import javax.inject.{Inject, Singleton}
 import models.User
-import models.errors.ErrorModel
-import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import services.{EmailVerificationService, VatSubscriptionService}
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.HeaderCarrierConverter
-import utils.LoggerUtil.{logDebug, logWarn}
-import views.html.email.VerifyEmailView
+import utils.LoggerUtil.{logDebug}
```

Comment:
Could you remove the curly brackets here as there's only one item in this import